data/korath/korath ships.txt

```diff
@@ -15,58 +15,58 @@ ship "Korath Dredger"
 	attributes
 		category "Heavy Freighter"
 		"cost" 29370000
-		"shields" 39000
-		"hull" 16000
-		"required crew" 201
-		"bunks" 395
+		"shields" 27000
+		"hull" 22000
+		"required crew" 137
+		"bunks" 357
 		"mass" 1503
-		"drag" 18.3
-		"heat dissipation" .65
+		"drag" 17.1
+		"heat dissipation" .45
 		"fuel capacity" 900
-		"cargo space" 605
-		"outfit space" 827
-		"weapon capacity" 297
-		"engine capacity" 181
-		
+		"cargo space" 675
+		"outfit space" 675
```

Comment:
```suggestion
		"outfit space" 783
```

I'm... OK with nerfing the outfit space somewhat, but not nearly this much. Given the nerfing of weapon capacity and the discussion about pushing this to be a "utility" or "support" ship... Well, let's let it have the space to actually be *good* at it.

Just to be clear, this isn't buffing the Dredger, it's merely reducing the severity of the nerf to its outfit space: With this it "only" loses 40 outfit space. I'd actually very much prefer to see it stay at it's current 827 space, but if we really must nerf it, losing 40 is acceptable.

Reply:
We did keep the weapon space nerfs for the most part so I think is okay, but I'd like to see one more response from @warp-core who though the outfit space nerf was needed before we continue.

Reply:
What about the extra 70 I added to cargo space?  I put that there specifically with the expectation that it could be used for outfit expansions.

Reply:
Honestly I do like that idea more, where we keep more of the nerfed outfit space and get more cargo to convert

Reply:
The cargo space was good, and the main reason I'm not pushing to have that outfit space nerf just zeroed out entirely.

That being said, the Korath Exiles operate primarily in two regions:
1. Near the Galactic Core
2. In the Ember Waste

While the core doesn't mechanically have any of the properties in does in Real Life, it theoretically is also a high-heat environment similar to the Ember Waste. As such, I think that having the Korath use expansions on a regular basis is a pretty bad idea. Combining unusually hot outfits with operating mostly in regions that are also unusually high-heat means that expansions will have a disproportionally higher impact on their ships than, say, a ship operating out in the cold  deeps of space.

Also worth noting that between the Xaplateaux and the map rework project, there is some push for the core to actually become something more akin to what it theoretically is in real life, which will make heat dissipation values more mechanically impactful than they are now too.

Reply:
The current weapon capacity is not really a whole lot lower than the previous value (276 vs 294), such a small decrease in outfit space (787 vs 827) would do little to push it away from an active combat role to a support one. What extra support is 787 instead of 675 outfit space going to allow it to provide?

Reply:
Mechanically speaking to avoid the heat dissipation effect of expansions, I'd rather remove the 70 cargo space you gave them, then fill the extra outfit space that this change has with cargo expansions. That'd work better, and not penalize their heat. Also ties into them being more flexible in terms of utility than the average ship.

Reply:
So to be clear Zitchas, you are putting your foot down on any nerf to outfit space at all?  Because with your suggestion, this is in fact a *buff* to outfit space.  With outfit expansions, that gives it one *more* outfit space, and 10 extra cargo.

Reply:
Well, this is the Korath. They live in space. So my first thought would be Hydroponics & food production! Unlike, say, a Republic fleet that launches from its home port with a finite set of supplies, goes out to do its patrol, and comes home; a Korath fleet is pretty much permanently in-flight. Sure, every fleet can't manufacture everything in it needs. But food and most essentials-of-life-consumables are probably at least *mostly* produced locally. 

There's already  been talk about having a range of "civilian" outfits that fulfill these roles. They might even have some benefits (like negative maintenance costs actually *paying* the player per day) that give players a reason to use them.

You are right, though, that right now, in vanilla, we don't have a lot of support capabilities. VitalChip has demonstrated healing weapons (although the AI has zero concept of how to use them) that could be very beneficial to this role. Likewise the possibility of carrying resupply munitions, replacement crew to replace lost ones (that's more of a lore than a mechanical thing though). There's also fuel regen & resupply. We're getting mechanics to allow carriers to serve as tankers that use fighters/drones to refuel ships in its fleet, so that's a potential major benefit right there. Being able to strip the fuel processors off of everyone else in the fleet to make other use of that space could be a very big help.

I guess when one gets right down to it, the only real way to have a "support" combat ship is to either load it up on weapons to contribute to the fight, or fill it with regen so that it can act like a shield to take some hits for the main combatants. Not ideal, I know. Some possibility that we could have a ship load up with jammers to reduce the ability of enemies to missile-spam. I'm not sure if jammers affect all missiles, or just missiles targeted specifically at the ship carrying them. But that *is* a possibility, and having a ship provide ECM capabilities for a fleet is also a very valuable capability.

Reply:
I *guess* we *might* be able to use that space if we *also* increased engine capacity... gave it more shielding and power than it actually needs....

And effectively just retaining its role as an overpowered battleship

Which, hey, I may not like, but I don't really care about the Dredger at all
...so long as this means you aren't going to fight me over my KWS diversification and expansion of use.

Reply:
In that regard, I'm still not in favor of buffing the outfit space back to this much for that very reason.

Reply:
At the very least I (and I'm sure warp as well) want to see a 100 or more nerf to outfit space, definitely not 40.

Reply:
That is the big problem, though. ES is primarily about ship combat. So "alternative Bactrian" boils down to being one of two options:
a) being a worse ship that no-one will use.
b) a ship that is equally combat capable but in a different way.

I firmly dislike option A, and am pushing to keep the Dredger somewhere in B where, in comparison terms, it loses out on some things like all the guns, but gains in other areas such as hp and overall ease-of-outfitting. 

And as far as that goes, I think we're pretty much there. The Dredger is, largely, going to be a less satisfying ship to fly with the massive (comparative) hit to mobility and the lack of direct-fire gun slots. We can at least give it the freedom/enjoyability of not forcing the player to struggle and scrimp for every ton of space required to mount extraneous outfits.

Reply:
> so long as this means you aren't going to fight me over my KWS diversification and expansion of use.

Very low chance of that happening. :) From what I've seen of your plans so far, I like them.

Reply:
So I suppose my real concern as far as the Dredger goes, is that if we keep it as this "do it all" ship... when I start putting forth larger, more specialised transports and battleships - the ships that are the entire reason I got in on this project - are you going to point to the Dredger and say, "No, those are too valuable, the Dredger can do it.  See, I even made this super contrived ground assault variant!"?  Because you've made comments to that effect already, and not really responded to my objections to them, and I'm starting to wonder if all the weeks of effort I've put into this were worth it.

Reply:
> > so long as this means you aren't going to fight me over my KWS diversification and expansion of use.
> 
> Very low chance of that happening. :) From what I've seen of your plans so far, I like them.

All right.  Sorry for getting my hackles raised there, GitHub didn't show me this before I finished.

Reply:
Sorry, I didn't mean it that way. When I was talking about World Ships, I was referring to them in their current incarnations as "The Big Bastions of Korath Civilization." When that role gets taken over by the Haven and other big ships, then the current ones are 100% fair game for general use as you have described. 

In that future scenario, I view things like the "Ground Assault Dredger" as being the "fast-response small ground attack transport" whereas the Transport WS would be for assaulting major targets, full-on ground invasions. (not merely an assasult or raid, but actual invade-and-occupy type stuff)

Reply:
It looks like you completely went over Warp's question/comment, but I expect another reply from him anyways as 40 seems way too low from what we suggested, and even lower than what you had asked to compromise with earlier.

Let me get this straight- we have nerfed outfit space. Somewhat. You bumped that up again past a value we don't necessarily agree on (to 276), from a value that was already perfectly fitting our weapon changes (242). Warp's compromise outfit value space was 675, which is still 75 tons more than what we had the initial nerfs at. Jumping from 674 back up to 783 seems *way* too much of a "compromise change" for what we are trying to do, especially if it's only a 40 ton nerf (and as Raven said, objectively a buff when it comes to using outfit expansions). 

Are you suggesting that we leave all of that lot of extra open outfit and weapon space for your suggested flexibility for the player? As far as that's concerned, that seems *against* the idea that Korath, a race currently in a bottleneck, are flying optimized ships that use almost every ton, which Raven is also preferred to having.

Reply:
No, it shouldn't be empty. It should, ideally, be filled with stuff that most humans never bother carrying on their ships but the Korath would almost certainly carry large volumes of (at least on a non-combat-centric-ship). The player, being human and not concerned with supporting a fleet that only touches land every few months at most and probably more like every few years, is perfectly justified in ripping it out to stuff more combat suitable stuff into its place.

We don't actually *have* those long-term-support outfits yet, so filling it with cargo expansions and probably a fuel processor is a good substitute.

Would increasing that help? Say, down to 763? (I'll change the original suggestion to that, as I'll be fine with that number)

Reply:
Again that still leaves: 

34 more extra weapon space from our nerf, used on nothing. I'm assuming *this* will be left open?

88 more tons of outfit space. Do you think all of the new Korath support outfits will weigh this much?

Reply:
You are right, the Korath need every ton they can get, so if a ship can handle having an extra bit of structure on there to hold stuff, they should be bolting it on. Just because we don't have an outfit to fill that space with right now doesn't mean that space isn't being used - it's just not being used by something that is relevant to the player. The player doesn't care about nurseries or hydroponics or weight-lifting rooms or medbays or common rooms or storage for personal effects. But I'm sure any "empty" space on a Korath ship is definiteliy being filled with stuff like these. (or their own culturally appropriate equivalents)

Reply:
I don't care about filling up all weapon capacity

I would like more engine capacity to use for reversers or other things.  Remember those are using up engine space now, too

Reply:
Ah, I see in your edit now. We probably shouldn't be thinking in terms of real life capacity as a lot of the game isn't based on that specifically, but I do appreciate you using a token amount. I would honestly be fine with the outfit space at 763. 

Still iffy on the weapon space, but so long as we keep the weapons where I placed them, it should be fine?

@warp-core Any thoughts?

Reply:
Sounds good to me. And yeah, I agree regarding the "not using real life capacities." Ships would look a *lot* different if that was the case. (and it'd reinforce the whole "almost everything we've got currently is basically varying sizes of interceptor and frigate ships")